content/courses/Ada_For_The_C_Embedded_Developer/chapters/02_Perspective.rst

```diff
@@ -148,17 +148,17 @@ The first line of the Ada code is giving us access to the :ada:`Ada.Text_IO`
 library which contains the :ada:`Put_Line` function we will use to print the
 text to the console. This is similar to C's :c:`#include <stdio.h>`. We then
 create a procedure which executes :ada:`Put_Line` which prints to the console.
-This is similar to C's :c:`printf` statement. For now, we can assume these Ada
+This is similar to C's :c:`printf` function. For now, we can assume these Ada
 and C features have similar functionality. In reality, they are very different.
 We will explore that more as we delve further into the Ada language.
 
 You may have noticed that the Ada syntax is more verbose than C. Instead of
 using braces :c:`{}` to declare scope, Ada uses keywords. :ada:`is` opens a
 declarative scope |mdash| which is empty here as there's no variable to
 declare. :ada:`begin` opens a sequence of statements. Within this sequence,
-we're calling the function :ada:`Put_Line`, prefixing explicitly by the name of
+we're calling the function :ada:`Put_Line`, prefixing explicitly with the name of
 the library unit where it's declared, :ada:`Ada.Text_IO`. The absence of the
-end of line :c:`\n` can also be noted, as :ada:`Put_Line` always terminates by
+end of line ``\n`` can also be noted, as :ada:`Put_Line` always terminates by
```

Comment:
Please not the change in syntax to render the newline. This will render it in a different "style" compared to the other inlined text which uses the :c:`` syntax.

Reply:
In general, we use language-specific syntax highlighting for any piece of code that might refer to source-code. For example, if we're referring to variable X in a previous code example, we'd write:
```
Variable :ada:`X` in the example above is [...]
```
We could apply the same principle here for `\n`. That being said, however, I don't have a strong opinion here. :-)

Reply:
Agree that this is the desired approach! However, when I tried using the :ada: and :c: syntax highlighting there seemed to be an issue escaping the ```\``` character.

Reply:
In that case, let's just use your change. :-)